specification/maps/data-plane/Search/preview/2021-11-01-preview/geocoding.json

```diff
@@ -125,7 +125,8 @@
     "/geocode": {
       "get": {
         "description": "**Geocoding**\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nIn many cases, the complete search service might be too much, for instance if you are only interested in traditional geocoding. Search can also be accessed for address look up exclusively. The geocoding is performed by hitting the geocoding endpoint with just the address or partial address in question. The geocoding search index will be queried for everything above the street level data. No POIs will be returned. Note that the geocoder is very tolerant of typos and incomplete addresses. It will also handle everything from exact street addresses or street or intersections as well as higher level geographies such as city centers, counties, states etc.",
-        "operationId": "GetGeocoding",
+        "operationId": "Get-Geocoding-(Preview)",
```

Comment:
Unclear why this is necessary.  This entire doc is preview, so it should be assumed that all APIs are preview.  Can you please clarify?

Reply:
this is to change the api title in the api doc.
![image](https://user-images.githubusercontent.com/86343602/144786177-38053cdc-4cfd-4a28-bf40-0b60f72e1ce4.png)
left hand side is the doc of this pr
right hand side is the doc without this pr

In the folder structure of this repo, we can clearly know it's a preview api. But this thing doesn't show in the api doc.
Thus, we want to change the title of api to tell users that this is a preview api.

Reply:
Thanks for the explanation.

@mikekistler is this the recommended way to achieve the desired doc comment?

Reply:
No, the `operationId` should not be used to mark the status of the operation.

The Azure Breaking Changes Policy does not allow changing the operationId even for new API versions.

Further -- I don't understand:
> We can clearly know it's a preview api by the folder structure. But it's not easy to tell in the api doc.

When I look at the doc, I can tell instantly that this is a preview API version:

![image](https://user-images.githubusercontent.com/85643503/144908177-5d935e80-f4a4-4208-b128-e3d713f4a060.png)

Reply:
I think we can ignore this breaking change violation because we haven't published this api.
You can take this as a 2nd PR of the swagger file.

As for the preview part, this is a request from our PM. We want to show the Preview wording at a conspicuous place.

Reply:
@chcmsft Once again -- the operationId is not the way to do this.  Once you release the API in preview, then from that point on the operationId cannot be changed.  So if it says "preview", it will say "preview" forever, even when you want to make the API GA.  That is obviously not right.

I suggest that you connect with the REST API docs team on your requirement. @kexugit is the PM and can help you here. You can also bring this issue to the weekly REST API docs office hours (Tuesdays at 12:30 PT) and you can get advice there.

Reply:
Hi @mikekistler, is there a document with the guidelines for that? I can see plenty examples across Azure API offerings with the "preview" word in the doc title, making it clear that it's a "Preview" offering. Furthermore, we've had recently other APIs moving from Preview to GA which included removing the "preview" word from the Doc Title.

Again, please let me know if you have any link to docs with best practices. It's a matter of making the fact that it is a public preview **clear and obvious** to the customer. It doesn't necessarily needs to be in the tittle, but just the version saying it doesn't seem like enough. We don't want customers putting Public Preview offerings in their production pipelines given they follow different SLAs from GA.

Reply:
Azure Breaking Changes policy is at http://aka.ms/AzBreakingChangesPolicy/.

You need to talk to the REST API Docs team -- as I said above -- to determine how to get "Preview" in the doc title.

But if it were me, I would resist the urge to put "Preview" in the doc title.  It is _plainly obvious_ from the API version on the page that the API is in preview.  We should leave it at that.  If some services decide to put "Preview" in the doc title, then that just creates confusion for customers looking at other services that have not done so.

But I'll let you duke that out with the REST API Docs folks.  Just don't use `operationId` to do it.

Reply:
Thanks Mike, will follow up with them. Do you have Ke Xu's alias so I can reach out?

Reply:
@chcmsft please incorporate the notes added by Jeff Sedlak, stating it's a preview, and remove the Preview word from the title so we can move forward with the documentation release.

Reply:
@mikekistler @jhendrixMSFT 
After move the preview reminder to the description, we don't have the preview in the `operationId` anymore.
But we still need to add `Search_` prefix to the `operationId` so that they would show in the search group.
Consider that this doc has not been published yet, I think we can ignore the breaking change due to the `operationId` now.
If there is no concern anymore, please help to review and merge.
Many thanks!

Reply:
@mikekistler is the breaking change ok?  Sounds like this hasn't been deployed yet.  If you can ack I'll approve and merge.